scalableminds/webknossos
# Scale organization Logo down if parent is small

### URL of deployed dev instance (used for testing):
- https://scaleorgalogo.webknossos.xyz

### Steps to test:
- open dataset view mode, change viewport sizes
- organization logo should be scaled down

------
